src/admin-views/attendees.php

```diff
@@ -11,31 +11,21 @@
 $singular = $pto->labels->singular_name;
 
 /**
- * Whether we should display the "Attendees" title.
+ * Whether we should display the "Attendees List" title.
  *
  * @since  4.6.2
  * @since  4.12.1 Append the post ID to the Attendees page title and each Ticket's name.
+ * @since  TBD Tweak wording from "Attendees" to "Attendees List" and change default from enabled to disabled.
  *
  * @param boolean                         $show_title
  * @param Tribe__Tickets__Tickets_Handler $handler
  */
-$show_title = apply_filters( 'tribe_tickets_attendees_show_title', true, $attendees );
+$show_title = apply_filters( 'tribe_tickets_attendees_show_title', false, $attendees );
```

Comment:
Would it make sense to use the `is_admin()` as the default?

```suggestion
$show_title = apply_filters( 'tribe_tickets_attendees_show_title', is_admin(), $attendees );
```